packages/imperative/src/plugins/cmd/list/list.definition.ts

```diff
@@ -21,5 +21,13 @@ export const listDefinition: ICommandDefinition = {
     type: "command",
     summary: "List installed plug-ins",
     description: "List all plug-ins installed.",
-    handler: join(__dirname, "list.handler")
+    handler: join(__dirname, "list.handler"),
+    options: [
+        {
+            name: "short",
+            aliases: ["-s"],
```

Comment:
Is there a reason this is -s instead of s?

Reply:
Good catch, fixed